addons/l10n_in_tcs_tds/data/account_tax_template_data.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_in.cess_sale_5" model="account.tax.template">
+        <field name="is_base_affected" eval="False"/>
```

Comment:
Maybe it is easier to change it right away in l10n_in

Reply:
okay

Reply:
@jco-odoo , I have moved these into l10n_in.